doc/cephfs/index.rst

```diff
@@ -4,30 +4,38 @@
  Ceph File System
 =================
 
-The Ceph File System (CephFS) is a POSIX-compliant file system that uses
-a Ceph Storage Cluster to store its data. The Ceph file system uses the same Ceph
-Storage Cluster system as Ceph Block Devices, Ceph Object Storage with its S3
-and Swift APIs, or native bindings (librados).
+The Ceph File System, or **CephFS**, is a POSIX-compliant file system built on
+top of Ceph's distributed object store, **RADOS**. CephFS endeavors to provide
+a state-of-the-art, multi-use, highly available, and performant file store for
+a variety of applications, including traditional use-cases like shared home
+directories, HPC scratch space, and distributed workflow shared storage.
+
+CephFS achieves these goals through the use of some novel architectural
+choices.  Notably, file metadata is stored in a separate RADOS pool from file
+data and served via a cluster of *Metadata Servers*, or **MDS**. Clients of the
+file system have direct access to RADOS for reading and writing file data
+blocks. For this reason, workloads may linearly scale with the size of the
```

Comment:
Do you also mention that the MDSes can be scaled to match the metadata work load?

Reply:
done